lib/galaxy/tool_util/deps/containers.py

```diff
@@ -372,7 +372,7 @@ def resolve(
         tool_info: "ToolInfo",
         index: Optional[int] = None,
         resolver_type: Optional[str] = None,
-        install: bool = True,
+        install: bool = False,
```

Comment:
This is causing a test failure, but it's necessary to make the functionality work, I'm not sure what we want the solution to be, I'm still unclear on the distinction on when `install` is intentionally set vs. `auto_install`, which is a resolver config option.

Reply:
I'm not sure how to phrase it better than you ? That's exactly the distinction (at the top level of the request). You don't wanna start pulling images when you just want to check what images can be resolved, which would be the effect. See the failing tests, where we asked which images can be resolved, and the cached resolver said "I got them!" when in fact it did not have them at the time of the request.